reports/docusign-autopopulation/index.md

```diff
@@ -0,0 +1,134 @@
+
+
+# DocuSign Auto-Populate Integration Set-up
+
+Skuid's [DocuSign connector] (https://docs.skuid.com/latest/en/data/docusign/#docusign) allows you to view and edit documents you've sent with DocuSign, but what if you want to send a DocuSign document from Skuid, filled with the data you've already pulled into your Skuid page? Luckily, Skuid can easily send requests via DocuSign's REST API to do exactly that. Follow these instructions to set up a document template in DocuSign and then connect to Skuid to send the document automatically populated with data from your Skuid page.
+ 
+## What you’ll need: 
+1.	A DocuSign sandbox account ([create one here](https://go.docusign.com/trial/)).
+2.	A Skuid page with data you want to push to a DocuSign template.
+3.	(Eventually) admin access to the production DocuSign account that will be sending the template document for signatures.
+
+## Set-up
+### DocuSign : 
+1.	In your sandbox account, navigate to the admin section. 
+2.	From the menu on the left, select “API and Keys”.
+3.	Click “Add Integrator Key”, define a description and save
+4.	Select your new integrator key and create the correct Skuid Redirect URIs (generate yours with [this tutorial](https://docs.skuid.com/latest/en/data/callback-urls-redirect-uris.html))
+5.	Generate a secret key and store it somewhere safe. (Warning: you will only be able to access it one time). Click Save. 
+6.	Go back to DocuSign 
+7.	Create a template and upload the document that is to be populated and sent 
+8.	Define recipients roles and signing order. (If the recipient is going to be variable, only fill in the role name)
+9.	Create signature tabs and define a role for each tab. 
+10.	Create plain text tabs (fields) for each data item that you want to populate. (Make sure to specify the recipient). 
+	- Define a custom data label (no spaces) for each tab. 
+11.	Take note of the Template Id (the Id can be found in the URL when viewing the document)
+ 
+
+### Skuid Configure: 
+1.	Go to Data Sources > Authentication Providers
+2.	Create a new Authentication Provider with a unique name and DocuSign Provider type. 
+3.	Set the Authorize Endpoint URL and Token Endpoint URL to: 
+	- Authorize: `https://account-d.docusign.com/oauth/auth`
+	-	Token: `https://account-d.docusign.com/oauth/token`
+4.	Set the Client Id to the Integrator Key Id defined in DocuSign
+5.	Set the Client Secret to the Secret Key you saved from DocuSign earlier. Save!
+6.	Now go to Data Sources > Data Sources
+7.	Create a New Data Source with a unique name and with the REST Data Source Type
+8.	Set the URL to  `https://demo.docusign.net/restapi` and select the Authentication Provider that you created in Steps 1-5.
+
+
+### Skuid Page:
+1.	Create a Model with REST as the data source type, and select the DocuSign data source that you created. Select Read/Write as the Model Behavior.
+2.	On this new model, select Methods>insert
+3.	Set values as follows: 
+	-	type = Insert
```

Comment:
We should use the actual model property names here to avoid confusion (and fix the indentation also): 

-	**Type**
-	**Data Source URL**
- **Data Source HTTP Verb**
- **Send new field values...**